Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rust:1.48 as builder
```

Comment:
Our pipelines/tests use latest stable Rust, maybe we should use latest stable here too? E.g. `FROM rust:latest` or `FROM rust:1`. Not sure how github actions docker cache will behave if we supply `rust:latest`, e.g. if the cache won't prevent us keeping up to date with stable Rust.

Alternatively, we should also run the pipelines/tests for the pinned version, as well as latest stable.

The rationale behind these two approaches is to make sure the packaged version is also always tested before it's released.

Reply:
Oh good point. Big oversight from me. 
Guess `rust:latest` should be fine. Haven't found much information on how long the images from Docker Hub get cached, just some vague general information from [here](https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows). It does not say anything about the Docker cache time but I doubt it would be very long given that so many pipelines depend on the `latest` tag. In the worst case we can change it back to `rust:1`.

Reply:
Makes sense, let's try that.